features/git-ship/current_branch/on_feature_branch/without_open_changes/same_user_different_email.feature

```diff
@@ -0,0 +1,34 @@
+Feature: git ship: shipping a feature branch where the main committer is the same user
+
+  As a developer shipping a feature branch under my secondary email
```

Comment:
That is obscured. It actually occurs one the step of setting up the commits. "remote" commits get created by creating commits in the secondary repository. I will make that more explicit